### DISCUSSION TITLE
Typo Correction | npx create-aptos-dapp , boilerplate template

### DISCUSSION THREAD

**Discussion by buzzlamicjihad** [aptos-labs/aptos-developer-discussions#310] [Discussions Feedback]
Hello,

I'm just poking around the developer documentation and giving the create aptos dapp templates a test run.

Although my development experience is very limited I am looking to get my feet wet using move rather than solidity or rust because I heard of nothing but good feedback from experienced devs.

I wanted to contribute in the best way I could but unfortunately I don't have much technical expertise, however I noticed something I can comment on, which is a small typo I came across.

<img width="777" alt="Screen Shot 2024-07-13 at 1 26 02 PM" src="https://github.com/user-attachments/assets/46f61a98-71b8-4dc0-b1ae-87000bda472a">

After I deploy the frontend following the [boilerplate template](https://aptos.dev/en/build/create-aptos-dapp/templates/boilerplate), the local site needs to correct "CAD Boilerplat Template" to "CAD **Boilerplate** Template.

This should be an easy fix.

**Comment by 0xmaayan** [comment]
Hi @buzzlamicjihad, thank you for reporting! If you want to try and tackle this issue, feel free to [open up a PR on the create-aptos-dapp repo](https://github.com/aptos-labs/create-aptos-dapp/blob/main/templates/boilerplate-template/frontend/components/Header.tsx#L6) with a fix - should be a very easy one.

If not, I would try find some time to fix it this week
